### DISCUSSION TITLE
SeleniumHQ /docker-selenium Xvfb-run don't manage vitual audio card

### DISCUSSION THREAD

**Discussion by azerty-sudo** [SeleniumHQ/docker-selenium#2279] [General]
Hello, 

I see that in the docker container you define a virtual audio card named "v1"

- docker-selenium/NodeBase/start-selenium-node.sh
- docker-selenium/Standalone/start-selenium-standalone.sh

However, I don't understand how the audio card is mapped, as it's not visible in the virtual screen.
How can I get this virtual audio card recognized in xvfb?
Because I'll need to send audio files to this virtual screen, can you explain pliz ?

docker-selenium/NodeBase/start-xvfb.sh

Cordially

**Comment by VietND96** [comment]
Can you describe on sending the audio file to virtual screen for which purpose?

**Reply by azerty-sudo** [reply]
Hello, 

To give you some context in the virtual screen, I'm opening a teams conversation with the browser and my aim is to play an audio file and on the remote host, record the audio to analyze it in relation to the input file.
The audio file is a recording of several seconds in "wav" format.
To send it, I use the "paplay" command with the "--device" option to send it to the v1 audio card and for the audio capture I use pacat

Cordially

**Reply by azerty-sudo** [reply]
The problem is that as dbus redirects to /dev/null even if we override this variable with another path when launching chrome for example we encounter dbus errors: ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket
So dbus doesn't work and we won't be able to retrieve the audio or microphone.
